### DISCUSSION TITLE
FileNotFoundError

### DISCUSSION THREAD

**Discussion by JabezBrew** [openai/whisper#485] [Q&A]
I've used every sort of file path- absolute and relative but I'm still getting a FileNotFoundError error.

**Comment by Majdoddin** [comment]
ffmpeg and/or ffmpeg-python are not installed, ffmpeg to your `PATH` environment variable.
https://github.com/openai/whisper/discussions/109

**Comment by JabezBrew** [comment]
I pip installed ffmpeg-python

**Reply by jjsai** [reply]
Did that work for you? I'm having the same issue, so I installed ffmpeg and ffmpeg_python but now I get the error "module 'ffmpeg' has no attribute 'Error'" any ideas? Thank You.

**Comment by jjsai** [comment]
I found this solution that worked for me: https://stackoverflow.com/questions/54262306/ffmpeg-python-wrapper-ffmpeg-run-getting-filenotfounderror. After following the instructions of how installing ffmpeg I was able to get the version using "ffmpeg -version." That was it. The code worked great.

**Reply by ChuckinAllen** [reply]
This helped thank you.
